packages/account/src/Components/address-details/__tests__/address-details.spec.js

```diff
@@ -262,4 +263,18 @@ describe('<AddressDetails/>', () => {
             expect(address_state_input.value).toBe('State 1');
         });
     });
+
+    it('should disable the field if it is immuatble from BE', async () => {
+        mock_props.disabled_items = ['address_line_1', 'address_line_2'];
+
+        render(<AddressDetails {...mock_props} />);
+
+        expect(screen.getByPlaceholderText(address_line_1)).toBeDisabled();
+        expect(screen.getByPlaceholderText(address_line_2)).toBeDisabled();
+        await act(async () => {
```

Comment:
is Async needed here? I do not see any await to resolve a promise within the following function.

Reply:
Replaced with await waitfor , Thanks for your feedback🫡